Gemfile

```diff
@@ -4,5 +4,5 @@ gem 'omnibus-software', git: 'git://github.com/datadog/omnibus-software.git', br
 gem 'httparty'
 gem 'win32-process'
 gem 'ohai'
-gem 'pedump', git: 'https://github.com/ksubrama/pedump', branch: 'patch-1'
+gem 'pedump'
```

Comment:
could you add a version constraint that reflects from which version the upstream gem includes the patch of the forked gem?